src/background.js

```diff
@@ -322,7 +322,29 @@ function stripFbclid(url) {
   return strippedUrl.href;
 }
 
+async function tabUpdateListener (tabId, changeInfo, tab) {
+  await updateBrowserActionIcon(tab.url);
+}
+
+async function updateBrowserActionIcon (url) {
+  if(isFacebookURL(url)) {
+    browser.browserAction.setPopup({popup: "./panel1.html"});
+    const FBC_STORAGE = await browser.storage.local.get();
```

Comment:
Global const's are generally uppercased where things are 100% constant. So like magic variables like `const ICON_COLOR = "red";` but `const fbcStorage = await ...` is more about preventing reassignment of  a dynamic variable.

Elsewhere is using `const url = new URL(options.url);` for example.